src/messaging/tests/TestReliableMessageProtocol.cpp

```diff
@@ -1244,7 +1220,6 @@ void CheckLostResponseWithPiggyback(nlTestSuite * inSuite, void * inContext)
         NL_TEST_ASSERT(inSuite, rm->TestGetCountRetransTable() == 0);
     }
 
-    // 1 tick is 64 ms, sleep 65*3 ms to trigger re-transmit from receiver
```

Comment:
The comment about how now we are trying to trigger retransmit from receiver should stay.